src/components/details/README.md

```diff
@@ -10,6 +10,26 @@ Component for conditionally revealing content, using the details HTML element.
 
 More information about when to use details can be found on [GOV.UK Design System](http://www.linktodesignsystem.com/details "Link to read guidance on the use of details on Gov.uk Design system website")
 
+## Quick start examples
+
+    <details class="govuk-c-details ">
+      <summary class="govuk-c-details__summary">
+        <span class="govuk-c-details__summary-text">Help with nationality</span>
+      </summary>
+      <div class="govuk-c-border govuk-c-border--left-narrow">
+        <div class="govuk-c-details__text">
+          <p>
```

Comment:
It'd be good to get the markup examples indented correctly.

Reply:
I'm going to open a new PR fixing the indentation of component examples.